utopia-remix/app/routes/v1.projectid.tsx

```diff
@@ -10,6 +11,9 @@ export async function loader(args: LoaderFunctionArgs) {
 
 export async function action(args: ActionFunctionArgs) {
   return handle(args, {
-    POST: proxy,
+    POST: {
+      handler: proxy,
+      validator: ALLOW,
```

Comment:
this should also validate the project access

Reply:
This route doesn't take a param - it's for creating (and returning) a new project ID, so there is nothing to validate

Reply:
abort abort

Reply:
this doesn't have a project yet, correct? ( this only creates a new id)